qiskit_experiments/framework/base_experiment.py

```diff
@@ -362,6 +359,27 @@ def circuits(self) -> List[QuantumCircuit]:
         # values for any explicit experiment options that affect circuit
         # generation
 
+    def _transpiled_circuits(self, backend: Backend) -> List[QuantumCircuit]:
```

Comment:
```suggestion
    def _transpiled_circuits(self) -> List[QuantumCircuit]:
```
The backend should come from `self` as in @yaelbh's PR

Reply:
Done.